docs-src/go/how-to-schedule-a-workflow-in-go.md

```diff
@@ -0,0 +1,233 @@
+This is useful when you need to execute a missed or delayed Action, or if you want to test the Workflow ahead of time.
+
+To backfill a Schedule in Go, use `Backfill()` on `scheduleHandle`.
+
+## Delete
+
+Deleting a Schedule erases a Schedule from a Workflow.
+Deletion does not affect any Workflows started by the Schedule.
+
+To delete a Schedule in Go, use the `Delete()` command on `scheduleHandle`.
+
+<!--SNIPSTART samples-go-schedule-create-delete {"selectedLines": ["42-49"]}-->
+
+[schedule/starter/main.go](https://github.com/temporalio/samples-go/blob/master/schedule/starter/main.go)
+
+```go
+// Delete the schedule once the sample is done
+defer func() {
+	log.Println("Deleting schedule", "ScheduleID", scheduleHandle.GetID())
+	err = scheduleHandle.Delete(ctx)
+	if err != nil {
+		log.Fatalln("Unable to delete schedule", err)
+	}
+}()
+```
+
+<!--SNIPEND-->
+
+## Describe
+
+You can retrieve information about the current Schedule configuration by using the `describe` action.
+This is helpful when you want to get a detailed view of the Schedule and its associated [Workflow Runs](/concepts/what-is-a-run-id).
+
+To describe a Schedule in Go, use `Describe()` on `scheduleHandle`.
+
+<!--SNIPSTART samples-go-schedule-unpause-describe {"selectedLines": ["115-118"]}-->
+
+[schedule/starter/main.go](https://github.com/temporalio/samples-go/blob/master/schedule/starter/main.go)
+
+```go
+description, err := scheduleHandle.Describe(ctx)
+if err != nil {
+	log.Fatalln("Unable to describe schedule", err)
+}
+```
+
+<!--SNIPEND-->
+
+## List
+
+When used, `list` returns all available Schedules and their Schedule Ids.
+
+To retrieve a list of Schedules, use `ScheduleClient().List()`.
+
+## Pause/unpause
+
+The `pause` and `unpause` actions enable starting or stopping all future Workflow Runs on a given Schedule.
+Pausing a Schedule halts all future Workflow Runs; unpausing a Schedule allows the Workflow to execute as planned.
+
+Pausing can be enabled when you create a Schedule by setting `State.Paused` to `true`.
+
+<!--SNIPSTART samples-go-schedule-update {"selectedLines": ["90-91"]}-->
+
+[schedule/starter/main.go](https://github.com/temporalio/samples-go/blob/master/schedule/starter/main.go)
+
+```go
+// Start the schedule paused to demonstrate how to unpause a schedule
+schedule.Description.Schedule.State.Paused = true
+```
+
+<!--SNIPEND-->
+
+Pausing can also be done by using `Pause()` on `scheduleHandle`.
+
+```go
+scheduleHandle.Pause()
+```
+
+To unpause a Schedule, use `Unpause()` on `scheduleHandle`.
+
+<!--SNIPSTART samples-go-schedule-unpause-describe {"selectedLines": ["107-110"]}-->
+
+[schedule/starter/main.go](https://github.com/temporalio/samples-go/blob/master/schedule/starter/main.go)
+
+```go
+// Unpause schedule
+log.Println("Unpausing schedule", "ScheduleID", scheduleHandle.GetID())
+err = scheduleHandle.Unpause(ctx, client.ScheduleUnpauseOptions{})
```

Comment:
Inconsistent that unpausing has logs and error checks and everything but pausing is just a single call w/ no error check